harper-core/src/linting/else_possessive.rs

```diff
@@ -18,8 +18,10 @@ impl Default for ElsePossessive {
             "everybody",
             "everyone",
             "nobody",
-            "noone",
-        ]);
+        ])
+        .or(Box::new(
+            SequencePattern::aco("no").then_whitespace().t_aco("one"),
```

Comment:
At the time I forgot there's also `ExactPhrase`, unless that's also case-sensitive?